### DISCUSSION TITLE
Cannot utilize Quad9 DNSCrypt in DNS settings

### DISCUSSION THREAD

**Discussion by kevindd992002** [AdguardTeam/AdGuardHome#5774] [Q&A]
I've tried using any of the Quad9 DNSCrypt stamps found [here ](https://dnscrypt.info/public-servers/) and the all don't work when doing a `Test Upstream`. I have no problems with Cisco DNSCrpyt servers though. How do I solve this?

**Comment by kevindd992002** [comment]
Anybody?

**Reply by amrmzr** [reply]
It happened to me as well but later I tried again and it worked. how many times you tried? is RAM usage is too high??
